packages/strapi-admin/admin/src/containers/InstalledPluginsPage/Row.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { IconLinks } from '@buffetjs/core';
+import { CustomRow } from '@buffetjs/styles';
+import { useGlobalContext, PopUpWarning } from 'strapi-helper-plugin';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useHistory } from 'react-router-dom';
+import { faTrash, faCog } from '@fortawesome/free-solid-svg-icons';
+import Text from '../../components/Text';
+import LogoContainer from './Logo';
+
+// TODO:
```

Comment:
Todo what ?

Reply:
When the media lib feature is merged we need to remove the upload link from this view as the settings will be accessible directly from the settings view